packages/devtools_app/lib/src/screens/vm_developer/vm_developer_common_widgets.dart

```diff
@@ -738,9 +737,9 @@ class VmServiceObjectLink extends StatelessWidget {
 
     final TextStyle style;
     if (isServiceObject) {
-      style = theme.fixedFontLinkStyle;
+      style = theme.linkTextStyle;
```

Comment:
Are these still using monospace fonts?

Reply:
No. Chrome DevTools does not use a monospace font for links either:
<img width="799" alt="Screenshot 2024-01-04 at 11 21 42 AM" src="https://github.com/flutter/devtools/assets/43759233/8e794d52-a2bb-4e96-b9c8-13fc6bd57bf4">